### PR TITLE
[ci] CoLoR has moved to github

### LINKED ISSUE #6194: [ci] Adapt to CoLoR moving to git.
As of today `CoLoR`'s CI is a bit of an odd animal as it is still hosted in a svn repository and we have to patch it manually.

However, CoLoR upstream has expressed their intention to move to git [and likely Github], so this issue is created to track the changes to the CI, as well as to be sure CoLoR upstream implements a branching strategy that we can use.

### PR DESCRIPTION
**Pull request by ejgallego**
Closes #6194 .

### CODE REVIEW

File: dev/ci/ci-color.sh [hunk 1]
```diff
@@ -3,33 +3,11 @@
...
-sed -i -e "6i From Coq Require Import FunInd." ${Color_CI_DIR}/Coccinelle/examples/cime_trace/more_list_extention.v
-sed -i -e "6i From Coq Require Import FunInd." ${Color_CI_DIR}/Coccinelle/examples/cime_trace/ring_extention.v
-sed -i -e "27i From Coq Require Import FunInd." ${Color_CI_DIR}/Coccinelle/list_extensions/dickson.v
-sed -i -e "26i From Coq Require Import FunInd." ${Color_CI_DIR}/Coccinelle/list_extensions/list_permut.v
-sed -i -e "23i From Coq Require Import FunInd." ${Color_CI_DIR}/Coccinelle/list_extensions/list_set.v
-sed -i -e "25i From Coq Require Import FunInd." ${Color_CI_DIR}/Coccinelle/list_extensions/list_sort.v
-sed -i -e "21i From Coq Require Import FunInd." ${Color_CI_DIR}/Coccinelle/list_extensions/more_list.v
-sed -i -e "21i From Coq Require Import FunInd." ${Color_CI_DIR}/Util/List/ListUtil.v
-sed -i -e "17i From Coq Require Import FunInd." ${Color_CI_DIR}/Util/Multiset/MultisetOrder.v
-sed -i -e "13i From Coq Require Import FunInd." ${Color_CI_DIR}/Util/Set/SetUtil.v
```
Comment (SkySkimmer) [inline dev/ci/ci-color.sh]: Is this sed stuff not necessary anymore?

Comment (ejgallego) [reply dev/ci/ci-color.sh]: Nop. Github's master has been updated to be compatible with 8.7/8.8.

Comment (ejgallego) [reply dev/ci/ci-color.sh]: We can also use regular overlays / PR to CoLoR now.

**Review by SkySkimmer — APPROVED** [review]

**Comment by Zimmi48** [comment]
And now only SF remains the odd one.

**Pull request merged**